src/components/Admin/AdminLogin/AdminLogin.tsx

```diff
@@ -0,0 +1,57 @@
+import { Form, FormLayout, TextField, Button, Card } from "@shopify/polaris";
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { authAccessToken } from "../../../utils/auth";
+
+export interface IAdminLoginProps {
+  setAccessSessionStateToken: React.Dispatch<
+    React.SetStateAction<string | null>
+  >;
+}
+
+export function AdminLogin({ setAccessSessionStateToken }: IAdminLoginProps) {
+  const { t } = useTranslation();
+  const [
+    shouldShowInvalidAccessToken,
+    setShouldShowInvalidAccessToken,
+  ] = useState(false);
+  const [accessToken, setAccessToken] = useState("");
+
+  const handleSubmit = () => {
+    const bearerToken = authAccessToken(accessToken);
```

Comment:
It is a convention for functions to have a verb prefix - `authAccessToken` sounds like a variable to me. `getAuthAccessToken` would have a clearer intent.